packages/discord.js/src/managers/GuildScheduledEventManager.js

```diff
@@ -44,24 +40,24 @@ class GuildScheduledEventManager extends CachedManager {
    * @property {string} name The name of the guild scheduled event
    * @property {DateResolvable} scheduledStartTime The time to schedule the event at
    * @property {DateResolvable} [scheduledEndTime] The time to end the event at
-   * <warn>This is required if `entityType` is `GuildScheduledEventEntityType.External`</warn>
+   * <warn>This is required if `entityType` is {@link GuildScheduledEventEntityType.External External}</warn>
```

Comment:
I’d say you don’t need the text, just leave the link itself

Reply:
Okay, I changed it.